deploy/crds/operators.coreos.com_servicebindings_crd.yaml

```diff
@@ -259,6 +260,8 @@ spec:
                 type: object
               type: array
           type: object
+          required:
+              - "services"
```

Comment:
They are actually optional, see: https://github.com/redhat-developer/service-binding-operator/blob/6d7a3aa2a396d7d02458a3f898807cdd71a0bae2/pkg/apis/apps/v1alpha1/servicebinding_types.go#L26

Reply:
Yes, but I could not find any other way to avoid using service and enforce use of services. Do you have anything in mind which I can try?

Reply:
Not sure what you are trying to solve in this PR. `service` fields are simply ignored now, also not part of the schema. Hence, why do we need to enforce anything?

Reply:
If we use the service field instead of services in SBR then the operator will crash at reconciling so to avoid happening that I'm trying to make sure we're using services not service.